### ISSUE TITLE
URL Preview for links

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockbox/lockbox-extension#605]
In standard web pages within Fx, a url preview is displayed in the bottom-left corner of the page on hover. Can we implement this for our links within Lockbox? 

<img width="446" alt="screen shot 2018-03-02 at 10 33 59 am" src="https://user-images.githubusercontent.com/6164801/36912887-6fda6dd6-1e05-11e8-99bd-bf60d67218b7.png">

**Comment by jimporter** [comment]
Not easily, no. All of our "links" are actually Redux actions that open a new window. (This is how we get telemetry for these links.) It's technically *possible* to fix this, but doing it in a maintainable way would require a fair amount of work.

**Comment by devinreams** [comment]
I also don't think this is possible, by design, on browser pages.

Another concrete example where this doesn't happen: if you hover on "FAQ" on our toolbar it _does not_ show the URL as `https://mozilla-lockbox.github.io/lockbox-extension/faqs/`.

**But** you don't get this if you go into about:addons either. For example, click into an extension and see the links on that view. For Lockbox the "Mozilla" name is linked, and the URL to github.com is, but neither show "previews" in the bottom of the browser. Suggesting to me this is intentional...

**Comment by jimporter** [comment]
On the other hand, open about:addons and go to "Get Add-ons" and look at the links for the authors of add-ons in that view. Those *do* have a tooltip.

Still, if we were actively using priorities on our issues (hint hint :)), I'd probably put this at P3.

**Comment by changecourse** [comment]
Agreed on prioritization, just wanted to document the (potential) issue and consider it for future efforts (perhaps long term future) :)
